arch/arm/soc/st_stm32/stm32l0/Kconfig.defconfig.stm32l072cz

```diff
@@ -0,0 +1,22 @@
+# Kconfig - ST Microelectronics STM32L072XB MCU
+#
+# Copyright (c) 2017 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32L072CZ
+
+config SOC
+	string
+	default stm32l072cz
+
+config FLASH_PAGE_SIZE
```

Comment:
Remove flash. It could be added it in a later PR

Reply:
Done